src/vaadin-text-field-mixin.html

```diff
@@ -471,20 +481,31 @@
       node.removeEventListener('blur', this._boundOnBlur);
     }
 
+    _addAutoselectListener(node) {
+      node.addEventListener('focus', this._boundAutoselect);
+    }
+
+    _removeAutoselectListener(node) {
+      node.removeEventListener('focus', this._boundAutoselect);
+    }
+
     ready() {
       super.ready();
 
       this._boundOnInput = this._onInput.bind(this);
       this._boundOnChange = this._onChange.bind(this);
       this._boundOnBlur = this.validate.bind(this);
+      this._boundAutoselect = this._autoselect.bind(this);
```

Comment:
I would suggest using `this._boundOnFocus` to align with other listeners here.

Reply:
I considered `focus`, but was a bit worried `_onFocus` would clash with something. I'll make the change, I agree it's much better, should have asked before :-)

Reply:
Done.